# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/syscalls.h-update.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- dietlibc-0.32/i386/syscalls.h	2008-09-30 23:02:35.000000000 +0200
+++ dietlibc/i386/syscalls.h	2009-12-02 03:22:03.000000000 +0100
@@ -325,7 +325,16 @@
 #define __NR_fallocate		324
 #define __NR_timerfd_settime	325
 #define __NR_timerfd_gettime	326
-
+#define __NR_signalfd4		327
+#define __NR_eventfd2		328
+#define __NR_epoll_create1	329
+#define __NR_dup3		330
+#define __NR_pipe2		331
+#define __NR_inotify_init1	332
+#define __NR_preadv		333
+#define __NR_pwritev		334
+#define __NR_rt_tgsigqueueinfo	335
+#define __NR_perf_event_open	336
 
 #define syscall_weak(name,wsym,sym) \
 .text; \
--- dietlibc-0.32/x86_64/syscalls.h	2008-09-30 23:02:35.000000000 +0200
+++ dietlibc/x86_64/syscalls.h	2009-12-02 03:22:03.000000000 +0100
@@ -334,6 +334,17 @@
 #define __NR_fallocate		285
 #define __NR_timerfd_settime	286
 #define __NR_timerfd_gettime	287
+#define __NR_accept4		288
+#define __NR_signalfd4		289
+#define __NR_eventfd2		290
+#define __NR_epoll_create1	291
+#define __NR_dup3		292
+#define __NR_pipe2		293
+#define __NR_inotify_init1	294
+#define __NR_preadv		295
+#define __NR_pwritev		296
+#define __NR_rt_tgsigqueueinfo	297
+#define __NR_perf_event_open	298
 
 #ifdef __PIC__
 #define syscall_weak(name,wsym,sym) \
